reek.yml

```diff
@@ -18,3 +18,4 @@ DuplicateMethodCall:
 TooManyStatements: 
   exclude: 
     - TransactionsController
+    - CategoriesController
```

Comment:
@marderer why did you exclude it?

Reply:
@AleksSenkou my actions in this controller have more than 5 line. And i can't do five. But may be better set 7 line for this option?